tests/test_kerns.py

```diff
@@ -577,6 +577,61 @@ def test(self):
         super().test()
 
 
+class TestChangePoints(GPflowTestCase):
```

Comment:
We're trying to move towards pytest (function) style tests instead of subclassing TestCase - this often makes tests much simpler, and will also make it easier to port to gpflow2, would be great if you could change these accordingly (please ask if you've got questions how or pointers to good examples)!

Reply:
Great stuff - yeah I've had a look at gpflow2 code (currently trying to port my generalised periodic kernel there) - looks like a great improvement over the current structure and design choices